package.json

```diff
@@ -57,14 +58,14 @@
         "content-type": "^1.0.4",
         "express": "^4.16.4",
         "fs-extra": "^7.0.1",
-        "iltorb": "^2.4.2",
+        "iltorb": "^2.4.3",
```

Comment:
Shall we include this library here? Maybe just in peerDeps?

Reply:
I think that it could be only a dev dependency. It is used in tests only.

Reply:
alright even better